es/utils/bignumber.ts

```diff
@@ -3,42 +3,32 @@
  * @module @aeternity/aepp-sdk/es/utils/bignumber
  * @example import { parseBigNumber, asBigNumber, isBigNumber, ceil } from '@aeternity/aepp-sdk/es/utils/bignumber'
  */
-import BigNumber from 'bignumber.js'
+import BigNumber from 'bignumber.js';
```

Comment:
`;` is not allowed by our linter rules, need to switch from `standard` to `ts-standard` for linting